src/Native/Windows/build-native.cmd

```diff
@@ -10,36 +10,53 @@ set __IntermediatesDir=""
 set __BuildArch=AnyCPU
 set __VCBuildArch=x86_amd64
 set CMAKE_BUILD_TYPE=Release
+set __EnvironmentSet=false
+set __VSVersion=vs2013
```

Comment:
> Why are we picking VS2013 version? For the managed side we have removed support for VS2013 so we should do the similar for the native side.

This is the default, but it may as well be set to an empty string. It is in the :ToolsVersion tag that we set the actual value via a check of the environment variables VisualStudioVersion/VS140COMNTOOLS/VS120COMNTOOLS. 

We use the higher version available but I can remove VS2013 support if that's the way of the managed side.